app/templates/components/article-wrapper.hbs

```diff
@@ -100,5 +100,8 @@
         {{/each}}
       {{/collapsible-menu}}
     {{/if}}
+    <div class="license-description wds-font-size-s wds-leading-tight">
```

Comment:
Don't you think it would be better to have all the styling in scss?

Reply:
for me it does not matter, the only difference I see is that it does not make css bigger ;)